final_task/MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include rss_reader\TimesNewRoman.ttf
+include rss_reader\VERSION.txt
```

Comment:
Because Linux and Windows have different filesystem separators, your setup installation will not work on Linux.